packages/components/src/Filters/FilterInput.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { Checkbox, Radio } from '@patternfly/react-core';
+
+export interface FilterInputProps {
+  addRemoveFilters?: (value?: string, param?: string, type?: string, checked?: boolean) => void;
+  className?: string;
+  currentPage?: string;
+  id: string;
+  label: React.ReactNode;
+  param: string;
```

Comment:
Based on the original implementation, the `param` prop should be optional.

Reply:
In Radio name is required

Reply:
Then we need an optional runtime type check. It is not true that the param is required for every case.